bats/fb-verify-packages.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:ts=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "verify no old SCL packages" {
+  run tSetOSVersion
+  run FOREMAN_VERSION=$(tForemanVersion)
+
+  if tIsRedHatCompatible; then
+    if [ $FOREMAN_VERSION == '1.20' ]; then
+      if [ `yum list installed | grep -q -E 'rh-ruby24|tfm-ror51'` ]; then
```

Comment:
@ekohl This actually worked, I just needed to keep it still wrapped in the if

Reply:
I guess the subshell is needed because of the pipe